### PR TITLE
feat(lp): adding new pattern for related resources

### PR DESCRIPTION
**Pull request by runrog**

### CODE REVIEW

**Comment by runrog** [comment]
[run cli]

**Comment by runrog** [comment]
[run clear]

**Comment by runrog** [comment]
[run ci]

File: styleguide/derek/pattern-components/lp-callout/_lp-callout.scss [hunk 1]
```diff
@@ -70,3 +70,78 @@
...
+
+.lp-legal-p {
+  font-size: 14px;
+  font-style: italic;
+}
+
+.lp-relatedResource-card-3 {
+  @include make-md-column(4);
+
+  .lp-relatedResource-img {
```
Comment (WidgetsBurritos) [inline styleguide/derek/pattern-components/lp-callout/_lp-callout.scss]: Can we a take more of a MaintainableCSS approach to this?  

`.lp-relatedResource-card3Img`

This seems like nesting we can avoid.

**Comment by WidgetsBurritos** [comment]
Since you're out today @runrog, we're gonna go ahead and just merge this in, and we'll correct the issues in our crash session today.

**Pull request merged**